src/backend/src/app.js

```diff
@@ -53,6 +52,19 @@ export async function init({ version, rootKey }) {
     coreStorage: () => new RAM(),
   })
 
+  await manager.start()
```

Comment:
No need to await this I don't think, getUrl methods will wait internally. We might want to catch errors on these start and stops, however the unhandled rejection handler will do that and probably suffice for now unless we have a different message we want to show to the user in this case.